tests/unresolved_bugs/test_bug_81.py

```diff
@@ -52,7 +52,12 @@ def build_automaton():
 
 
 @pytest.mark.skipif(not on_linux, reason="Works only on linux")
-def test_does_not_leak_memory():
+def does_not_leak_memory():
+    # This is problematic, some memory leaks are still there
+    # so this needs to be investigated and re-enabled after fixing
```

Comment:
This was already in `[tests/unresolved_bugs](https://github.com/WojciechMula/pyahocorasick/pull/194/commits/b7d67e25dad29abe71f1449d77a33a44c9b622c5#diff-954a1ceddd469ad81f17135ad46ed7609eb84e8aa3f7a66da7e6ccfb2b2ee93a)` and was causing issues on mac/windows before, now the same is happening on linux sometimes. And the memory leak issue is already tracked so disabling this.